code/controllers/subsystem/triumph/buy_datums/communal/longer_week.dm

```diff
@@ -0,0 +1,20 @@
+/datum/triumph_buy/communal/preround/longer_week
+	name = "Longer Week"
+	desc = "Contribute to extend the working week (round) by 2 whole daes (40 mins) at minimum! Automatically refunds if it does not reach its goal before the round starts."
+	triumph_buy_id = TRIUMPH_BUY_LONGER_WEEK
+	maximum_pool = 100
+
+/datum/triumph_buy/communal/preround/longer_week/on_activate()
+	. = ..()
+	SSmapping.add_world_trait(/datum/world_trait/longer_week, 0)
+	SSticker.time_until_vote += 40 MINUTES
```

Comment:
this is a lot of time for only 100 triumphs

Reply:
100 is a lot for normal players